packages/js-utils/src/is-nullish.ts

```diff
@@ -0,0 +1,13 @@
+function isNullish( value: any ): boolean {
```

Comment:
We can do better than this for the types 😀 

```suggestion
function isNullish( value: any ): value is null | undefined {
```

Reply:
Thanks, I'll take the suggestion 👍